Dockerfile

```diff
@@ -15,25 +15,60 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-ARG VERSION=latest
-FROM rero/rero-ils-base:${VERSION}
-
-USER 0
-
-COPY ./ ${WORKING_DIR}/src
-WORKDIR ${WORKING_DIR}/src
-COPY ./docker/uwsgi/ ${INVENIO_INSTANCE_PATH}
-
-RUN chown -R invenio:invenio ${WORKING_DIR}
-
-USER 1000
+FROM python:3.6-slim-buster
 
 # Set last GIT commit hash. Example:
 # docker build --build-arg GIT_HASH=$(git rev-parse @) .
 ARG GIT_HASH
 # Last GIT commit hash
 ENV INVENIO_RERO_ILS_APP_GIT_HASH ${GIT_HASH:-''}
 
+# Install Invenio
+ENV WORKING_DIR=/invenio
+ENV INVENIO_INSTANCE_PATH=${WORKING_DIR}/var/instance
+RUN mkdir -p ${INVENIO_INSTANCE_PATH}
+
 ENV INVENIO_COLLECT_STORAGE='flask_collect.storage.file'
-RUN ./scripts/bootstrap --deploy && rm -fr ui/node_modules
 
+# create user
+RUN useradd invenio --uid 1000  --home ${WORKING_DIR}
+
+# copy uwsgi config files
+COPY --chown=invenio:invenio ./docker/uwsgi/ ${INVENIO_INSTANCE_PATH}
+
+# copy everything inside /src
+RUN mkdir -p ${WORKING_DIR}/src
+COPY --chown=invenio:invenio . ${WORKING_DIR}/src
+WORKDIR ${WORKING_DIR}/src
+
+# for some files to be written (example: egg-info)
+RUN chown -R invenio:invenio ${WORKING_DIR} \
+  && chmod -R go+w ${WORKING_DIR}
+
+# require debian packages
+# Set locales to C.UTF-8 to avoid some problem:
+# Cf. https://click.palletsprojects.com/en/5.x/python3/#python-3-surrogate-handling
+RUN set -ex \
+  && buildDeps=' \
```

Comment:
&& doing the same as `set -e` maybe we can use ; instead

Reply:
This is done.